media/css/m24/copy.scss

```diff
@@ -6,7 +6,7 @@
 .m24-c-copy {
     box-sizing: border-box;
     font-size: $alias-text-title-h4;
-    margin: 0 auto;
+    margin: 0 auto $spacer-xl;
```

Comment:
**Issue:** This adds too much space under the other copy groups on the page (ie the product list).  Can you add it to the top of the grid-container instead? Or add it conditionally with a separate class?